pages/common/vzdump.md

```diff
@@ -0,0 +1,28 @@
+# vzdump
+
+> Backup Utility for virtual machines and containers.
+> More information: <https://pve.proxmox.com/pve-docs/vzdump.1.html>.
+
+- Dump a guest virtual machine into default dump directory (usually `/var/lib/vz/dump/`), no snapshot:
```

Comment:
Perhaps this will read better?
```suggestion
- Dump a guest virtual machine into the default dump directory (usually `/var/lib/vz/dump/`), excluding snapshots:
```